File: PHP/UberPool.php
<?php
    require_once('Car.php');
    
    class UberPool extends Car{
        private $brand;
        private $model;

        // CONTRUCTORES
        public function __construct($license,$driver,$brand,$model){
            parent::__construct($license,$driver); // asi se declara la SUPER CLASE
            $this->brand = $brand;
            $this->model = $model;
        }

        // GETTERS AND SETTERS
        public function getBrand(){
            return $brand;
        }
        public function setBrand($brand){
            $this->brand = $brand;
        }
        public function getModel(){
            return $model;
        }
        public function setModel($model){
            $this->model = $model;
        }

        // METODOS
        


    }

?>

File: Java/UberPool.java
public class UberPool extends Car{
    private String brand;
    private String model;
    
    // CONSTRUCTOR
    public UberPool(String license,Account driver,String brand,String model){
        super(license, driver);
        this.brand = brand;
        this.model  = model;    
    }

    // GETTERS AND SETTERS
    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }
    
    

    
}
